client/rest/src/main/java/org/elasticsearch/client/DeadHostState.java

```diff
@@ -53,6 +53,11 @@ private DeadHostState() {
         this.failedAttempts = previousDeadHostState.failedAttempts + 1;
     }
 
+    DeadHostState(int failedAttempts, long deadUntilNanos) {
```

Comment:
I'm not sure about this - it helps with testing but I expect it conflicts with something @javanna is doing now.